src/FSharp.Markdown/Markdown.fs

```diff
@@ -28,19 +30,19 @@ type MarkdownColumnAlignment =
 /// Represents inline formatting inside a paragraph. This can be literal (with text), various
 /// formattings (string, emphasis, etc.), hyperlinks, images, inline maths etc.
 type MarkdownSpan =
-  | Literal of string
-  | InlineCode of string
-  | Strong of MarkdownSpans
-  | Emphasis of MarkdownSpans
-  | AnchorLink of string 
-  | DirectLink of MarkdownSpans * (string * option<string>)
-  | IndirectLink of MarkdownSpans * string * string
-  | DirectImage of string * (string * option<string>)
-  | IndirectImage of string * string * string
-  | HardLineBreak
-  | LatexInlineMath of string
-  | LatexDisplayMath of string
-  | EmbedSpans of MarkdownEmbedSpans
+  | Literal of string * MarkdownRange option
```

Comment:
(This will be a great improvement in general, because it'll be possible to extract information about nodes based on the names, but this is only available in F# since 2015 (I think?)).